pkg/operator/webhook_configuration.go

```diff
@@ -21,54 +22,58 @@ const genericCABundleInjectorLabel = "service.beta.openshift.io/inject-cabundle"
 // it will be created. If it does exist, the metadata of the required
 // mutatingwebhookconfiguration will be merged with the existing mutatingwebhookconfiguration
 // and an update performed if the mutatingwebhookconfiguration spec and metadata differ from
-// the previously required spec and metadata. For further detail, check the top-level comment.
-func applyMutatingWebhookConfiguration(client dynamic.Interface, recorder events.Recorder,
+// the previously required spec and metadata based on generation change.
+func applyMutatingWebhookConfiguration(client v1.MutatingWebhookConfigurationInterface, recorder events.Recorder,
```

Comment:
I would suggest just using `kubernetes.Interface` and then making into a `MutatingWebhookConfigurationInterface` within the code itself